src/utils/xrSE_Factory/ai_space.h

```diff
@@ -34,10 +34,24 @@ class CAI_Space
         EVENT_SCRIPT_ENGINE_RESET,
         EVENT_COUNT,
     };
-    CEventNotifierCallback::CID Subscribe(CEventNotifierCallback* cb, EEventID event_id) { return 0; }
+
+    template <class CB, class... Args>
+    CEventNotifierCallback::CID Subscribe(EEventID event_id, Args&&... args)
+    {
+        return 0;
+    }
+
     bool Unsubscribe(CEventNotifierCallback::CID cid, EEventID event_id) { return true; }
 };
 
+// Static initializer for every translation unit
+// Need to avoid static initilization order problem
+static struct SAI_Space_Initializer
+{
+    SAI_Space_Initializer();
+    ~SAI_Space_Initializer();
+} aiSpaceInitializer;
```

Comment:
According to the code style global variable name must start with capital letters.
```suggestion
} AISpaceInitializer;
```